spec/orm/activerecord_spec.rb

```diff
@@ -1195,6 +1197,14 @@ def monkey
     end
 
     describe "#remote_images_urls=" do
+      before do
+        sham_rack_app = ShamRack.at('www.example.com').stub
+        sham_rack_app.register_resource('/test.jpg', File.read(file_path('test.jpg')), 'images/jpg')
```

Comment:
I'd much rather extract this into a support module which provides a macro. Perhaps `setup_remote_url(url, file, mime)`, or perhaps `mime` can be derived from `file` automatically.